packages/react-components/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@fluentui/react-components",
+  "version": "9.0.0-alpha.0",
+  "description": "Suite package for converged React components",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.2",
+    "@fluentui/scripts": "^1.0.0",
+    "@types/react": "16.9.42",
+    "@types/react-dom": "16.9.10",
+    "react": "16.8.6",
+    "react-dom": "16.8.6"
+  },
+  "dependencies": {
+    "@fluentui/react-avatar": "^0.12.0",
+    "@fluentui/react-button": "^1.0.0-beta.35",
+    "@fluentui/react-image": "^0.8.0",
+    "@fluentui/react-link": "^1.0.0-beta.43",
+    "@fluentui/react-make-styles": "^0.2.8",
+    "@fluentui/react-provider": "^0.2.0",
+    "@fluentui/react-theme": "^0.3.3",
+    "@fluentui/react-theme-provider": "^1.0.0-beta.27",
+    "@fluentui/set-version": "^8.0.0-beta.2",
+    "tslib": "^1.10.0"
+  },
+  "peerDependencies": {
+    "@types/react": ">=16.8.0 <18.0.0",
+    "@types/react-dom": ">=16.8.0 <18.0.0",
+    "react": ">=16.8.0 <18.0.0",
+    "react-dom": ">=16.8.0 <18.0.0"
+  },
+  "beachball": {
+    "tag": "alpha",
```

Comment:
Still need to agree on whether this should be published under `latest` or `alpha`.

Reply:
Agreed, it will be `alpha`.